.changeset/olive-papayas-wink.md

```diff
@@ -0,0 +1,7 @@
+---
+'polaris-for-figma': patch
+---
+
+docs(polaris-for-figma/README.md): Fix yarn script
+
+Fix polaris-for-figma/readme.md
```

Comment:
OK, this should be the final thing: 😓 

This will go into a long list of improvements that will eventually be visible on the Polaris site. This description doesn't really explain what the fix is or how it makes things better. Could you perhaps change it to something like?

> Improves the readme to better explain development and build workflows.

Thanks!

Reply:
@martenbjork thanks, 

Done ✅